sendgrid/sendgrid.py

```diff
@@ -32,7 +32,8 @@ class SendGridAPIClient(BaseInterface):
     def __init__(
             self,
             api_key=None,
-            host='https://api.sendgrid.com',
+            host=None,
+            region='global',
```

Comment:
I'd vote for keeping region as an optional param and defaulting host to `api.sendgrid.com`. It keeps the behaviour same for users.